src/bpftrace.h

```diff
@@ -85,6 +87,15 @@ struct HelperErrorInfo
   location loc;
 };
 
+struct PerfEventCallbackCookie {
+  PerfEventCallbackCookie(BPFtrace* bpftrace, int cpu)
+  : bpftrace(bpftrace)
+  , cpu(cpu)
+  {}
+  BPFtrace* bpftrace;
+  int cpu;
+};
+
```

Comment:
my previous design, https://github.com/iovisor/bpftrace/pull/1357, coordinated map lookups between the BPF program and userspace. `cpu` was used to determine where to locate string content in the per-CPU map.

this change isn't required any more, but I kept it because it seemed like the correct way to expose CPU in case a future feature has the same needs (userspace lookup of entry from a per-CPU map when only the key is known). should I remove it for now?

Reply:
removed to make the review smaller.